cavestory.txt

```diff
@@ -0,0 +1,9 @@
+[Section]
+Name = Cave Story
+Version = 1.0.0.6
+License = Freeware
+Description = A Japanese freeware PC side-scrolling platformer game.
+Category = 4
+URLSite = https://www.cavestory.org/
+URLDownload = https://www.cavestory.org/downloads/cavestoryen.zip
```

Comment:
Could you please add the sha1 hash of the zip file?
That way we can see when the zip file is replaced with a version that is not tested.
Preferrably, link to a version that stays 'stable'

Reply:
It has been done. Do you want a link to a video of it?